app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -166,14 +170,20 @@ class RegisterController @Inject() (val helpToSaveService:     HelpToSaveService
     result.fold({ e ⇒
       logger.warn(s"Could not get enrolment status or session: $e")
       internalServerError()
-    }, {
-      _.flatMap(_.accountNumber).fold(SeeOther(routes.EligibilityCheckController.getCheckEligibility().url)){
+    }, { session ⇒
+      session.flatMap(_.accountNumber).fold(SeeOther(routes.EligibilityCheckController.getCheckEligibility().url)){
         accountNumber ⇒
-          Ok(views.html.register.account_created(accountNumber))
+          val email = session.flatMap(_.confirmedEmail).fold("")(_.toString)
```

Comment:
we probably don't want to show the page with an empty email. If there is no email the session is in a bad state - we're redirecting to check eligibility above so it'd make sense to do the same here. It'd make sense to have something like:
```scala
val x: Option[String,String] = for{
   s <- session
   a <- s.accountNumber
   e <- s.email
  } yield a -> e

x.fold(...)(...)
```

Reply:
ok thank you, I have updated the PR